docs/source/tutorials/automate_training.rst

```diff
@@ -0,0 +1,170 @@
+Optimize Hyperparameters
```

Comment:
Really clear documentation, great workt!

Reply:
Thanks!

Reply:
Really like it, thank you @ahill187 !